cms/djangoapps/contentstore/course_info_model.py

```diff
@@ -93,7 +93,7 @@ def update_course_updates(location, update, passed_id=None, user=None, request_m
         track_course_update_event(location.course_key, user, course_update_dict)
 
         # send course update notification
-        if ENABLE_COURSE_UPDATE_NOTIFICATIONS.is_enabled(location.course_key):
+        if ENABLE_NOTIFICATIONS.is_enabled(location.course_key):
```

Comment:
Remove waffle flag condition. Notification signal checks `ENABLE_NOTIFICATIONS.is_enabled(course_id)` before sending notification. This line is redundant.